.agents/skills/e2e-nl/SKILL.md

```
@@ -0,0 +1,276 @@
+allowed-tools: Read, Write, Edit, Bash, Grep, Glob, Agent
+---
+
+# E2E Natural-Language Spec Generator
+
+You produce a natural-language e2e test specification for an API
+endpoint from its code. Output format follows `references/format.md`,
+modeled on the worked examples in `references/examples.md`.
+
+This skill is **step 1 in a pipeline**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)